<launch>
    <param name="/use_sim_time" value="true" />

    <arg name="vision" default="true" />

    <param name="sim" value="true" />

    <param name="q_imunominal_imu_w" value="1.0" />
    <param name="q_imunominal_imu_x" value="0.0" />
    <param name="q_imunominal_imu_y" value="0.0" />
    <param name="q_imunominal_imu_z" value="0.0"/>

    <param name="q_dvl_auv_w" value="0.0" />
    <param name="q_dvl_auv_x" value="1.0" />
    <param name="q_dvl_auv_y" value="0.0"/>
    <param name="q_dvl_auv_z" value="0.0" />

    <param name="auv_dvl_offset_x" value="0.0" />
    <param name="auv_dvl_offset_y" value="0.0" />
    <param name="auv_dvl_offset_z" value="-0.3" />

    <include file="$(find bringup)/launch/bringup.launch">
        <arg name="sim" value="true" />
        <arg name="vision" value="$(arg vision)" />
    </include>

    <include file="$(find sim)/launch/endpoint.launch"></include>

    <node name="unity_bridge" pkg="sim" type="unity_bridge.py" output="screen"/>

</launch>